doc/rst/usingchapel/chplenv.rst

```diff
@@ -489,17 +490,17 @@ CHPL_GMP
 CHPL_HWLOC
 ~~~~~~~~~~
    Optionally, the ``CHPL_HWLOC`` environment variable can select between
-   no hwloc support or using the hwloc package distributed with Chapel in
-   third-party.  Note that hwloc is only used by the qthreads tasking layer,
-   and does not need to be built for other tasking layers.  Current options
-   are:
+   no hwloc support, using the hwloc package distributed with Chapel in
+   third-party, or using a system jemalloc.
 
-       ======= ==============================================================
-       Value   Description
-       ======= ==============================================================
-       none    do not build hwloc support into the Chapel runtime
-       hwloc   use the hwloc distribution bundled with Chapel in third-party
-       ======= ==============================================================
+       ======== ==============================================================
+       Value    Description
+       ======== ==============================================================
+       none     do not build hwloc support into the Chapel runtime
+       system   use a system install of hwloc
```

Comment:
@mppf I was thinking about this over the weekend and discussing a little bit with Greg. 

What do you think about not publicly documenting `CHPL_HWLOC=system` support, and instead putting it in a technote for now? My concern is that I don't really want to encourage users to use an arbitrary "system" install of hwloc (or jemalloc.) I know we support `CHPL_GMP=system`, but I think that's different because GMP's public API has been widely supported and stable for years, whereas jemalloc/hwloc are changing pretty frequently.  I only really want to support our bundled versions (and if we get a debian package, whatever the default package there is), but I don't really want to encourage users to install/use their own versions or use a a system install or use a system install on a platform we don't test.